.github/workflows/examples.yml

```diff
@@ -2,6 +2,7 @@ name: pytest for examples
 
 on:
   pull_request:
+    #    branches: [ master ]
```

Comment:
not needed?

Reply:
no,  see also line 6 (below).
I deleted line 5 anyways in the last commit